devel/docker-credential-helper-osxkeychain/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/docker/docker-credential-helpers 0.6.4 v
+revision            0
+name                docker-credential-helper-osxkeychain
+categories          devel
+platforms           darwin
+license             Apache-2
+maintainers         {emcrisostomo @emcrisostomo} \
+                    openmaintainer
+description         Docker credential helper for macOS
+long_description    docker-credential-helpers is a suite of programs to use \
+                    native stores to keep Docker credentials safe.  This \
+                    port installs the macOS keychain credential helper.
+
+checksums           rmd160  53f9fc31d4f31c219c3226abe6cac9f2a79a02ce \
+                    sha256  a3a3e785931533b3cd868d555d251d3ee185dc63671bfb71cd790f12bfd40796 \
+                    size    35654
+
+set docker.helper   docker-credential-osxkeychain
+build.cmd           ${go.bin} build -ldflags -s -o bin/${docker.helper} osxkeychain/cmd/main_darwin.go
```

Comment:
Modifying the `build.cmd` works. An alternate way is to leave `build.cmd` unchanged and use `pre_args` and `args`:

```
build.pre_args           -ldflags -s -o bin/${docker.helper}
build.args               osxkeychain/cmd/main_darwin.go
```